OffsetLib/wftel.py

```diff
@@ -0,0 +1,46 @@
+import time
+import argparse
+
+import ktl
+
+import KeckLogger
+
+parser = argparse.ArgumentParser(description="Wait for the telescope move to complete, stores time it took to complete",
+                         usage="wftel.py")
+
+args = parser.parse_args()
+
+def wftel():
+    dcs = ktl.Service('dcs')
+    autresum = dcs.read('autresum')
+    startTime = time.time()
+    axestat = dcs.monitor('AXESTAT')
+    ktl.waitfor(axestat == "tracking")
+    active = dcs.read("AUTACTIV")
+    if (active == 'no'):
+        print("WARNING: guider not currently active.\n")
+        return
+    count = 0
+    while(True):
```

Comment:
I think we can do this without the count loop.  I did something similar for the instruments, though I'm not sure it is the best format for this. Here's my version of this:
```
def wftel(timeout=20):
    startedat = datetime.utcnow()
    endat = startedat + timedelta(seconds=timeout)
    autresum = ktl.cache(service='dcs', keyword='AUTRESUM')
    autresum0 = autresum.read()
    autresum.monitor()

    axestat = ktl.cache(service='dcs', keyword='AXESTAT')
    axestat.monitor()
    ktl.waitfor(axestat == "tracking")

    autactiv = ktl.cache(service='dcs', keyword='AUTACTIV')
    autactiv.monitor()
    if autactiv == 'no':
        log.warning("Guider not currently active.")
        return

    while dt.utcnow() < endat and autresum == autresum0:
        time.sleep(0.2)
        if dt.utcnow() >= endat:
            log.warning(f'Timed out waiting for AUTRESUM')

    autgo = ktl.cache(service='dcs', keyword='AUTGO')
    autgo.monitor()
    while autgo not in ['RESUMEACK', 'GUIDE']:
        time.sleep(0.2)
        if dt.utcnow() >= endat:
            log.warning(f'Timed out waiting for AUTGO')

    elapsedtime = datetime.utcnow() - startedat
    return elapsedtime
```

I find this way of creating the keyword, then setting it to monitor a bit inelegant, so I'm open to other ways, but the loops waiting for keywords to reach a value are more compact and easier to follow.  I also think we can use short sleeps since we don't want to wake seconds if we can help it.